ansible/templates/nginx.conf

```diff
@@ -70,12 +70,21 @@ server {
         proxy_cache_min_uses    5;
     }    
 
-    location /pictshare/ {
+    # Redirect pictshare images to pictrs
+    rewrite ^(/pictshare/.*)$ /pictrs/image/$1;
+
+    # pict-rs images
+    location /pictrs/image/ {
       proxy_pass http://0.0.0.0:8537/;
       proxy_set_header X-Real-IP $remote_addr;
       proxy_set_header Host $host;
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 
+      # Block the import
+      location /pictrs/import {
+        return 403;
+      }
+
       if ($request_uri ~ \.(?:ico|gif|jpe?g|png|webp|bmp|mp4)$) {
         add_header Cache-Control "public, max-age=31536000, immutable";
       }   
```

Comment:
Remove this, pictrs already sends a cache-control header and having it twice breaks things.